camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -343,6 +343,7 @@ def __init__(self, boards, flags=0, fisheye_flags = 0, pattern=Patterns.Chessboa
         self.pattern = pattern
         self.br = cv_bridge.CvBridge()
         self.camera_model = CAMERA_MODEL.PINHOLE
+        self.declare_parameter('vga_scale', 0)
```

Comment:
`Calibrator` is not a ROS Node, this code is incorrect. You can check some failures in the CI

Reply:
If that's the case, I assume it's more practical to make the `scale`, a class member opposed to a function param since it's not required and would break API